### ISSUE TITLE
Graphical SSH connection can't take a SSH key

### ISSUE THREAD

**Issue by alx359** [Opticos/GWSL-Source#143]
When trying to connect to a remote server with the _Graphical SSH Connection_, there's a field to add an `SSH Private PPK Key` but it isn't working for me. The error says:

`Unable to use key file ""C:/Users/<USER>/.ssh/id_rsa_conv.ppk"" (unable to open file)
Using username "root".
Access denied
`
Note how there are redundant double quotes in the path

Also tried with the private key file alone - `id_rsa.ppk` and got the same error.

The `settings.json` section seems okay:
 "putty": {
  "ip": "<IP>",
  "ssh_key": "C:/Users/<USER>/.ssh/id_rsa_conv.ppk"
 },

That aforementioned key works fine with Putty standalone and WinSCP.
Checked GWSL according the manual and all seems okay my end.
Please advise, thanks.

**Comment by Pololot64** [comment]
Hey @alx359 . I think it is a bug on my end. I will try to fix in the code. In the meanwhile, GWSL can still work with putty.

**Comment by alx359** [comment]
Yes, it works with a direct passw. Would you want me to test a fix?

**Comment by Pololot64** [comment]
If you would like. It may be fixed in the source already I don't know.

**Comment by alx359** [comment]
What do I need to do? What file(s) need change?

**Comment by Pololot64** [comment]
The very end of manager.py

**Comment by alx359** [comment]
Sorry, ain't familiar with the code. `https://github.com/Opticos/GWSL-Source/blob/master/manager.py` is said to be updated like 5 month ago. Is that the file that I need to update locally, and where does it go in a W10 machine?

**Comment by Pololot64** [comment]
It can run anywhere as long as it is relatively near the other files. I would maybe test by seeing what putty cmd arguments work to launch it with keys
